modules/vaos/app/controllers/vaos/v2/eps_appointments_controller.rb

```diff
@@ -27,30 +27,12 @@ def show
       # @param appointment_data [Hash] Raw appointment data from the EPS service
       # @return [Eps::EpsAppointmentSerializer] Serialized appointment with referral and provider data
       def assemble_appt_response_object(appointment)
-        referral = fetch_referral(appointment)
         provider = fetch_provider(appointment)
-        eps_appointment = VAOS::V2::EpsAppointment.new(appointment, referral, provider)
+        eps_appointment = VAOS::V2::EpsAppointment.new(appointment, provider)
```

Comment:
Note: This is the only spot referral and provider are used, so we can safely remove all references to referral.